### ISSUE TITLE
Sendoff should parse on last valid record delimiter

### ISSUE THREAD

**Issue by kmorrisongr** [pechersky/sendoff#26]
`SDBlock.records()` stripping removes any text that is after the first `>` encountered. While I do not believe `sendoff` should "support" malformed SD files that have `<>` in record names or values, I do think it should "leave it as it found it".

With a simple loop on the included `mwe.sdf` (renamed to `.txt` for Github):
```
    for sdblock in SDBlock.from_lines(input):
        sdblock.write(output)
```
nothing gets changed.

But with this loop:
```
    for sdblock in SDBlock.from_lines(input):
        for record, _ in sdblock.records():
            print(record)
            break
        break
```
The record name `Sulfamid_>NSO2N<` becomes `Sulfamid_`.

I suspect the change required is here https://github.com/pechersky/sendoff/blob/main/sendoff/sdblock.py#L104-L106. Maybe a `r">$"` or some other regex?

[mwe.txt](https://github.com/pechersky/sendoff/files/7864555/mwe.txt)

**Comment by pechersky** [comment]
I will put test cases for a similar molecule that has a `record, value = ("Rec>ord", "Value)`. In the meantime, you should know that RDKit itself will convert that data, as a dict via `mol.GetPropsAsDict()` to `{"Rec": "Value"}`.

**Comment by pechersky** [comment]
Closed by #32
